usaspending_api/settings.py

```diff
@@ -26,6 +26,9 @@
 DOWNLOAD_DB_TIMEOUT_IN_HOURS = 4
 CONNECTION_MAX_SECONDS = 10
 
+# Value is set per environment using formula: ( TOTAL_MEMORY * 0.25 ) / MAX_CONNECTIONS
```

Comment:
Could you add an extra line that states that `MAX_CONNECTIONS` here is the max connections that the DB serving downloads is expected to see (which may be different than the DB serving api requests) @sethstoudenmier